packages/apps/graph/src/graph/query/chain-non-fungible-account.ts

```diff
@@ -0,0 +1,51 @@
+import { COMPLEXITY } from '@services/complexity';
+import { getAccountDetails } from '@services/node-service';
+import { getTokenDetails } from '@services/token-service';
+import { normalizeError } from '@utils/errors';
+import { builder } from '../builder';
+import ChainNonFungibleAccount from '../objects/chain-non-fungible-account';
+import { ChainNonFungibleAccountName } from '../types/graphql-types';
+
+builder.queryField('chainNonFungibleAccount', (t) =>
+  t.field({
+    description: 'Retrieve an account by its name on a specific chain.',
+    args: {
+      accountName: t.arg.string({ required: true }),
+      chainId: t.arg.string({ required: true }),
+    },
+    type: ChainNonFungibleAccount,
+    nullable: true,
+    complexity:
+      COMPLEXITY.FIELD.CHAINWEB_NODE +
```

Comment:
I would do this one times 2, since it's two calls to chainweb node. Nothing to prisma

Reply:
The token details calls prisma and account details calls chainweb-node